src/components/FormValidator.js

```diff
@@ -54,6 +54,11 @@ export default class FormValidator {
     }
   }
 
```

Comment:
**Could be improved**

You could make a special method `resetValidation` for clearing errors and controlling the submit button.
It could look something like this:

```js

resetValidation() {
      this._toggleButtonState(); <== controlling the submit button ==

      this._inputList.forEach((inputElement) => {
        this._hideError(inputElement) <== clearing errors ==
      });

    }



```

And you can call it in `index.js` when you click on open buttons